GoogleForCreators/web-stories-wp
# Remove Elements Tab Experiment

## Context



This PR removes the Elements Tab Experiment from the repo and the subsequently associated files.

## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



This removes the Elements Tab Experiment from the Experiments in Stories.

## Testing Instructions






This PR can be tested by following these steps:
1. Navigate to `Stories > Experiments`
2. The Elements Tab experiment should be gone
3. You should also have no Elements tab available in the Insert mini menu in the Editor.

For reference, the previous option and tab looked like this (Thanks Pascal for the following screenshots):
<img width="414" alt="Screenshot 2022-04-05 at 13 18 36" src="https://user-images.githubusercontent.com/841956/161744853-5815009a-f484-4836-a43e-04ebeff4a6ab.png">

<img width="491" alt="Screenshot 2022-04-05 at 13 18 49" src="https://user-images.githubusercontent.com/841956/161744848-d6741a09-7d3c-40f9-a64a-74f275ca798d.png">

## Reviews

### Does this PR have a security-related impact?


No.

### Does this PR change what data or activity we track or use?


No.

### Does this PR have a legal-related impact?


No.

## Checklist




---



Fixes #11186
